### ISSUE TITLE
ping response protocol 

### ISSUE THREAD

**Issue by jocawtho** [Azure/azure-functions-host#3178]
#### Investigative information

Please provide the following:

- Timestamp: ongoing
- Function App version (1.0 or 2.0-beta): both versions
- Function App name: hml-products-cs-dii (or other apps being pinged by the scale controller experiencing this exception)
- Region: all

#### Repro steps

Provide the steps required to reproduce the problem:

Issue started with 6/19 release
Not sure how to repro, it might just be on unhealthy apps. I found a symptomatic app and rdp'd to the worker to investigate. Using netmon, i found that pings to the root of the site were returning "Host is not running" or some such message. This causes a protocol exception with the 204 http status code in the scale controller.
from NetMon: "HTTPPayloadLine: Function host is not running." 

#### Expected behavior

Provide a description of the expected behavior.

There should be no http response with a 204 status code.

#### Actual behavior

Provide a description of the actual behavior observed.

from NetMon: "HTTPPayloadLine: Function host is not running." 

#### Known workarounds

Provide a description of any known workarounds.

This isnt "broken" the ping still gets through. It just causes thousands and thousands of errors in the scale controller

**Comment by jocawtho** [comment]
Hey Paul, pinging this again. Did we have a reason for this yet?
Thanks :)

**Comment by fabiocav** [comment]
@jocawtho are you still seeing this? This issue got lost in the triage process.

**Comment by jocawtho** [comment]
Yes, This is still the most noisy error we see in the scale controller
